doc: subcommand workflows (#69)
nightwatchjs/mobile-helper-tool: docs/subcommands/uninstall.md

## Workflows of the `uninstall` subcommand

**Syntax**
```sh

> npx @nightwatch/mobile-helper android uninstall [flags|cliConfigs]

Reviewer: ```suggestion
npx @nightwatch/mobile-helper android uninstall FLAG [configs]
```

```

### 1. Delete an Android Virtual Device

> Run the below command to delete a currently present AVD:

Reviewer: ```suggestion
Run the below command to delete an AVD:
```
